vllm/model_executor/layers/linear.py

```diff
@@ -34,7 +34,6 @@ def create_weights(self, layer: torch.nn.Module, layer_name: str,
                        output_sizes_per_partition: List[int], input_size: int,
                        output_size: int, params_dtype: torch.dtype,
                        **extra_weight_attrs) -> Dict[str, Any]:
-      
```

Comment:
can this be removed? will cause an extra rebase for no reason

Reply:
they are from format.sh .. but I can revert it and we can do a format.sh on the final branch.